frontend/src/concepts/pipelines/content/createRun/contentSections/ParamsSection/IntegerInputParam.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { NumberInput } from '@patternfly/react-core';
+import { InputParamProps } from './types';
+
+export const IntegerInputParam: React.FC<InputParamProps> = ({ id, value, onChange }) => {
+  const [integerValue, setIntegerValue] = React.useState<number | ''>(parseInt(String(value)) || 0);
+
+  const onMinus = (event: React.MouseEvent<Element>) => {
+    const newValue = (integerValue || 0) - 1;
+    setIntegerValue(newValue);
+    onChange(event, newValue);
+  };
+
+  const onPlus = (event: React.MouseEvent<Element>) => {
+    const newValue = (integerValue || 0) + 1;
+    setIntegerValue(newValue);
+    onChange(event, newValue);
+  };
+
+  return (
+    <NumberInput
+      id={id}
+      value={integerValue}
+      onMinus={onMinus}
+      onChange={(event) => {
+        const { value: targetValue } = event.target as HTMLInputElement;
+
+        setIntegerValue(targetValue === '' ? targetValue : +targetValue);
+        onChange(event, targetValue);
+      }}
+      onPlus={onPlus}
+      minusBtnAriaLabel="minus"
+      plusBtnAriaLabel="plus"
+      min={0}
```

Comment:
why is the min 0?

Reply:
Because I don't think it would be sense to have a negative value here. Do you think we should allow for negative values?

Reply:
we should not restrict a dynamic number input to only positive when KF can accept a negative number as input.

Reply:
yeah KF allows it